docs/usage/managed_seed.md

```diff
@@ -0,0 +1,74 @@
+# Register Shoot as Seed
+
+An existing shoot can be registered as a seed by creating a `ManagedSeed` resource. This resource replaces the `use-as-seed` annotation that was previously used to create [shooted seeds](shooted_seed.md). It contains:
+
+* The name of the shoot that should be registered as seed.
+* An optional `seedTemplate` section that contains the `Seed` spec and parts of its metadata, such as labels and annotations.
+* An optional `gardenlet` section that contains:
+    * `gardenlet` deployment parameters, such as the number of replicas, the image, etc.
+    * The `GardenletConfiguration` resource that contains controllers configuration, feature gates, and a `seedConfig` section that contains the `Seed` spec and parts of its metadata.
+    * Additional configuration parameters, such as the garden connection bootstrap mechanism (bootstrap token, service account, or none), and whether to merge the provided configuration with the configuration of the parent `gardenlet`.
+    
+Either the `seedTemplate` or the `gardenlet` section must be specified, but not both: 
+
+* If the `seedTemplate` section is specified, `gardenlet` is not deployed to the shoot, and a new seed is registered based on the template.
+* If the `gardenlet` section is specified, `gardenlet` is deployed to the shoot, and it registers a new seed upon startup based on the `seedConfig` section of the `GardenletConfiguration` resource.
+
+Note the following important aspects:
+
+* Unlike the `Seed` resource, the `ManagedSeed` resource is namespaced. Currently, managed seeds are restricted to the `garden` namespace.
+* The newly created `Seed` resource always has the same name as the `ManagedSeed` resource. Attempting to specify a different name in `seedTemplate` or `seedConfig` will fail. 
+* The `ManagedSeed` resource must always refer to an existing shoot. Attempting to create a managed seed referring to a non-existing shoot will fail.
+* A shoot that is being referred to by a managed seed cannot be deleted. Attempting to delete such a shoot will fail.  
```

Comment:
```suggestion
* A shoot that is being referred to by a `ManagedSeed` cannot be deleted. Attempting to delete such a shoot will fail.  
```

Reply:
Just to clarify: I use "managed seed" when talking about the clusters, similarly to the way "shoot" and "seed" are often used (and "shooted seed" was used in the past). I use "`ManagedSeed`" when specifically talking about the resource. In this particular case it's the resource so I will change as you suggest.

Reply:
Done